care/facility/models/mixins/permissions/facility.py

```diff
@@ -18,21 +18,42 @@ def has_write_permission(request):
             return (
                 request.user.is_superuser
                 or (
-                    request.user.user_type <= User.TYPE_VALUE_MAP["LocalBodyAdmin"]
+                    request.user.user_type > User.TYPE_VALUE_MAP["DistrictAdmin"]
```

Comment:
Regular users won't be able to create facilities now right? was that a requirement?